### PR TITLE
refactored some functions

### PR DESCRIPTION
**Pull request by cem313**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -7,21 +7,17 @@ let baseUrl = "https://pokeapi.co/api/v2/pokemon";
 let page = 1;
 
 function returnQueryParameters(num) {
-  // return `?offset=${(num - 1) * 20}`;
```
Comment (jmw1493) [inline app.js:10]: why'd you remove this line?

Comment (jmw1493) [inline app.js:10]: Why'd you remove this line?

**Review by jmw1493 — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (cem313) [reply app.js:10]: the pokemon made me!

Comment (jmw1493) [reply app.js:10]: ok got it

**Review by jmw1493 — APPROVED** [review]

**Pull request merged**